File: src/codeblocks/julia/about/3j5a.jl
"Java JavaScript Julia" |> lowercase |> collect |> 
                        chs -> reduce(function(dict, ch) haskey(dict, ch) ? (dict[ch] += 1; dict) : dict end, 
                                      chs, init = Dict('j' => 0, 'a' => 0)) |> 
                        collect |> 
                        jaPairs -> map(jaPair -> string(jaPair[2], jaPair[1]), jaPairs) |> 
                        join

File: src/codeblocks/javascript/about/3j5a.js
"Java JavaScript Julia".toLowerCase().split("")
                       .reduce((a, b) => a[b] !== undefined ? (a[b] += 1, a) : a, 
                        {j: 0, a: 0, _: function() {return `${this.j}j${this.a}a`}})
                       ._(); 